deployments/helm/KubeArmorOperator/README.md

```diff
@@ -1,28 +1,33 @@
-## Install KubeArmorOperator
-Install KubeArmorOperator using the official `kubearmor` Helm chart repo.Also see [values](#Values) for your respective environment.
-```
+# Install KubeArmorOperator
+
+Install KubeArmorOperator using the official `kubearmor` Helm chart repo. Also see [values](#values) for your respective environment.
+
+```bash
 helm repo add kubearmor https://kubearmor.github.io/charts
 helm repo update kubearmor
-helm upgrade --install kubearmor-operator kubearmor/kubearmor-operator -n kube-system
+helm upgrade --install kubearmor-operator kubearmor/kubearmor-operator -n kubearmor --create-namespace
 ```
 
 Install KubeArmorOperator using Helm charts locally (for testing)
-```
+
+```bash
 cd deployments/helm/KubeArmorOperator
-helm upgrade --install kubearmor-operator . -n kube-system
+helm upgrade --install kubearmor-operator . -n kubearmor --create-namespace
 ```
 
 ## Values
+
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
 | kubearmorOperator.name | string | kubearmor-operator | name of the operator's deployment |
 | kubearmorOperator.image.repository | string | kubearmor/kubearmor-operator | image repository to pull KubeArmorOperator from |
 | kubearmorOperator.image.tag | string | latest | KubeArmorOperator image tag |
 | kubearmorOperator.imagePullPolicy | string | IfNotPresent | pull policy for operator image |
```

Comment:
Can we update this table with the new values?

Reply:
The values seems to be matching the current state defined at Helm `values.yaml`. Can you please define what's need to be updated here?

Reply:
We added `configSpec` to values.yaml, right? Shouldn't we put in this table what it is for?

Reply:
The configSpec defined in `values.yaml` is the same as [KubeArmorConfig specification](https://github.com/kubearmor/KubeArmor/tree/main/deployments/helm/KubeArmorOperator#kubearmorconfig-specification)

Reply:
Agreed. But how are users supposed to know if there is a `configSpec` variable which can be used to specify `KubeArmorConfig` in the first place!